modules/overlay.nix

```
@@ -1,7 +1,14 @@
-{ lib, pkgs, ... }:
+{ lib, ... }:
 
 {
   nixpkgs.overlays = [
     (import ../overlay.nix)
   ];
+
+  assertions = [
+    {
+      assertion = lib.versionAtLeast lib.version "23.11";
+      message = "Jovian NixOS currently requires Nixpkgs 23.11 or newer to build.";
```

The actual suggestion should be to track nixos-unstable, rather than a specific version number, I think.

```suggestion
      message = "Jovian NixOS currently requires tracking nixos-unstable, (from 23.11 or newer) to build.";
```

We will invariably soon be tracking 24.whatever whenever nixos-unstable and our usage of it breaks on 23.11.

---

We can always just increase the number if we start requiring a newer version. Though given we mostly all test against unstable...